geoprocessing/src/modules/planning-units/planning-units.controller.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Controller,
+  Logger,
+  Get,
+  Req,
+  Param,
+  UploadedFile,
+  Post,
+  UseInterceptors,
+} from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+
+import { apiGlobalPrefixes } from 'src/api.config';
+import { uploadOptions } from 'src/utils/file.utils';
+import { ShapeFileService } from '../shapefiles/shapefiles.service';
+
+@Controller(`${apiGlobalPrefixes.v1}/planning-units`)
+export class PlanningUnitsController {
+  private readonly logger: Logger = new Logger(PlanningUnitsController.name);
+  constructor(public shapefileService: ShapeFileService) {}
```

Comment:
Consider injecting the logger, otherwise it won't leverage global settings (like muting logs)

Reply:
Following the project's approach. Should we consider DI-n all the loggers in a refactor task?

Reply:
We don't have to postpone this very piece until "refactor" and starting doing it "right" now ; )

Reply:
@alexeh for context, Kamil and I discussed this in a different PR and converged on injecting this. In general, your is a valid concern - we certainly don't want to be introducing a different way of doing many small things for each piece of code/service, so it is always a good idea to plan an alignment of existing code when we change how we do something in a new component. In this case, you would not have been aware of the DI discussion for loggers, of course.
Given this, in this case it's ok to start injecting the logger here since it's a small change, and we'll take care of aligning the rest in a separate PR (feel free to create a task and take it or bid for takers!).

Reply:
Changes applied [here](https://github.com/Vizzuality/marxan-cloud/pull/161/commits/1cf47128646b7c495378b79d8e2dc76ba563eb83)